privacyidea/static/components/user/controllers/userControllers.js

```diff
@@ -25,19 +30,102 @@ angular.module("privacyideaApp")
                                                gettextCatalog){
 
         $scope.formInit = {};
+
         ConfigFactory.getEditableResolvers(function (data){
             var resolvers = data.result.value;
             var resolvernames = [];
+            var userAttributes = [];
             for (var rname in resolvers) {
                 resolvernames.push(rname);
             }
             $scope.formInit.resolvernames = resolvernames;
             if (resolvernames.length > 0) {
                 $scope.resolvername = resolvernames[0];
             }
+
+            userAttributes = getUserAttributes(data);
+            $scope.userAttributes = userAttributes;
+            $scope.getAddUserAttributes($scope.resolvername);
+
         });
 
+        function getUserAttributes(data) {
+            var resolvers = data.result.value;
+            var allResolverAttributes = [];
+
+            for (var rname in resolvers) {
+                var resolver = resolvers[rname];
+                switch (resolver.type){
+                    case "ldapresolver":
+                        var userinfo = JSON.parse(gettextCatalog.getString(resolver.data.USERINFO));
```

Comment:
You added this to be displayed in translation. But in my case no translation was available.
Then it is prefixed with "[Missing]: password".
JSON.parse will interpret the missing as an array, which causes the translation to fail.
I will think of something here...